osmosis/assetlist.json

```diff
@@ -25755,6 +25755,41 @@
       "name": "Earn CDT Vault",
       "display": "earnCDT",
       "symbol": "earnCDT"
-    }
+    },
+    {
+      "description": "The Spice memecoin",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1n6asrjy9754q8y9jsxqf557zmsv3s3xa5m9eg5/uspice",
+          "exponent": 0
+        },
+        {
+          "denom": "uspice",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "factory/osmo1n6asrjy9754q8y9jsxqf557zmsv3s3xa5m9eg5/uspice",
+      "name": "Spice",
+      "display": "Spice",
+      "symbol": "Spice",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/spice.png",
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/spice.png",
```

Comment:
Please replace the images with a version that's <250kB

Reply:
I hate to request this, but can we go a smidge smaller still? The 250kB limit may be hardcoded into validation, and it would be best to limit having to manually resize a bunch of images at that point. Ideally under 200kB but that 'ideal' recommendation is a bit arbitrary.

Reply:
ofcourse no worries :+1: